internal/sgbouncewizard/sgbouncewizard.go

```diff
@@ -313,40 +311,48 @@ func (srv *Server) ChangelogContext(next http.Handler) http.Handler {
 
 		limit, ok := queryParams["limit"]
 
-		if len(limit) > 1 {
-			paramError := errors.New("Duplicate Parameters")
+		if !ok || len(limit) > 1{
+			paramError := errors.New("Invalid limit Parameter: does not exist or to many")
+			http.Error(w, paramError.Error(), http.StatusBadRequest)
+			return
+		}
+
+
+		offset, ok := queryParams["offset"]
+		if !ok ||  len(offset) > 1 {
+			paramError := errors.New("Invalid offset Parameter: does not exist or to many")
 			http.Error(w, paramError.Error(), http.StatusBadRequest)
+			return
+		}
+
+
+		limitAsInt, err := strconv.Atoi(r.FormValue("limit"))
```

Comment:
instead of `limitAsInt` just call it `limit`... no reason to put variable types in the names